metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def sbcape_cin(pressure, temperature, dewpt, profile=None):
```

Comment:
We can see what @dopplershift thinks here, but I don't think we need a `profile` argument. Duplicate the work of calculating a parcel in this as it's a simple wrapper.